src/ryr-email-templatizer.html

```diff
@@ -33,20 +33,19 @@
       _renderTemplate(templateVars) {
         let template =
           `
-Dear city of Austin,
+          Dear city of Austin,
 
-I would like to have a bike rack here: {{rackLocation | shout |blank>_not provided_}}.
+          I would like to request a bike rack at {{Address}}, Geolocation  {{rackLocation}}. The business supporting this demand is {{businessName}}, located at {{businessAddress}}.
 
-{{if businessName}}The business supporting this demand is {{businessName | blank>_not provided_}}{{if businessAddress }}, located at {{businessAddress | blank>_not provided_}}.{{else}}.{{/if}}{{/if}}
+          {{if contactName}}The point of contact is {{contactName | blank>_not provided_}} {{if contactPhone}}and can be contacted via phone ({{contactPhone}}) {{/if}}{{if contactEmail}}or email ({{contactEmail | blank>_not provided_}}).{{/if}}{{/if}}
```

Comment:
As you put `{{if contactEmail}}`, we are certain to have the `contactEmail` information and the filter `{{contactEmail | blank>_not provided_}}` will never be used (i.e. it will never display `_not provided_`).